src/theme/ContactForm/form-container.js

```diff
@@ -59,13 +60,11 @@ const FormContainer = (props) => {
         <img {...getImageProps('bgCircleLeft', 'bg-circle-whole.svg')} />
         <img {...getImageProps('bgCircleBottom', 'bg-circle-half.svg')} />
         <img {...getImageProps('bgCircleRight', 'bg-circle-whole.svg')} />
-        <div className="mainContainer documentContainer postContainer">
+        <div className="mainContainer documentContainer postContainer width-wrapper">
```

Comment:
Where are the rules for this class?

Reply:
They were in the shared libra-docusaurus-plugins repo Layout/styles.css, but I should move them to universal.css since they're being used in more than just Layout.

```
.width-wrapper {
  margin-left: auto;
  margin-right: auto;
  max-width: 1440px;
  width: 100%;
}
```